rancid-tomatillos/src/api.js

```diff
@@ -0,0 +1,36 @@
+const api = {
```

Comment:
Good call moving all of these to a helper file, really cleans up the components to not have api calls in them :)